portal-ui/src/screens/CreatePassword/index.tsx

```diff
@@ -0,0 +1,269 @@
+// This file is part of MinIO Kubernetes Cloud
+// Copyright (c) 2020 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import Typography from '@material-ui/core/Typography';
+import {makeStyles} from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Copyright from "../../components/Copyright";
+import request from "superagent";
+import {useHistory} from "react-router";
+import {CircularProgress} from "@material-ui/core";
+import storage from 'local-storage-fallback';
+
+const useStyles = makeStyles(theme => ({
+    '@global': {
+        body: {
+            backgroundColor: theme.palette.common.white,
+        },
+    },
+    paper: {
+        marginTop: theme.spacing(8),
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    avatar: {
+        margin: theme.spacing(1),
+        backgroundColor: theme.palette.secondary.main,
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(3),
+    },
+    submit: {
+        margin: theme.spacing(3, 0, 2),
+    },
+    errorBlock: {
+        color: 'red',
+    },
+    spinner: {
+        margin: 'auto',
+
+    }
+}));
+const CreatePassword: React.FC = () => {
+    const classes = useStyles();
+    const {push} = useHistory()
+    const [password, setPassword] = React.useState<string>('');
+    const [token, setToken] = React.useState<string>('');
+    const [tokenValidated, setTokenValidated] = React.useState<boolean>(false);
+    const [tokenValid, setTokenValid] = React.useState<boolean>(true);
+    const [repeatPassword, setRepeatPassword] = React.useState<string>('');
+    const [error, setError] = React.useState<string>('');
+
+    const formSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+
+        if (password != repeatPassword) {
```

Comment:
Similarly, we are getting an error for not having double equals here:

```
  Line 78:22:   Expected '!==' and instead saw '!='  eqeqeq
```

Can we apply the following change to get rid of it?

```suggestion
        if (password !== repeatPassword) {
```